src/devices/Mcp23xxx/Bank.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Mcp23xxx
+{
+    public enum Bank
```

Comment:
Forgive my ignorance, but what does bank mean on this context? Is it just a logical collection of registers?

Reply:
You are correct.  The group either by port (which is only in 16-bit registers) or by type (e.g. IODIRA, IODIRB, etc.).  This is just a way to help support finding the correct register address.

Reply:
I see, adding a xml (meaning a `///` comment) comment to the enum would be nice in order for the consumers to get the info on the popup snippet

Reply:
Updated with comments.